src/main/java/dev/openfeature/sdk/Value.java

```diff
@@ -14,7 +17,7 @@
 @ToString
 @EqualsAndHashCode
 @SuppressWarnings({"PMD.BeanMembersShouldSerialize", "checkstyle:MissingJavadocType"})
-public class Value {
+public class Value implements Cloneable{
```

Comment:
```suggestion
public class Value implements Cloneable {
```

This is the reason for pr pipeline failure.